.kokoro/docs/publish.sh

```diff
@@ -10,6 +10,15 @@ elif [ "$#" -ne 0 ]; then
     exit 1;
 fi
 
+# Run publish-reporter-script to report back the status in release PR.
+CURRENT_DIR=$(dirname "${BASH_SOURCE[0]}")
+REQUIREMENTS_FILE=$(realpath "${CURRENT_DIR}/requirements.txt")
+python3.9 -m venv venv
+source venv/bin/activate
+python3.9 -m pip install --require-hashes -r "${REQUIREMENTS_FILE}"
+python3.9 -m releasetool publish-reporter-script > /tmp/publisher-script; source /tmp/publisher-script
+deactivate
```

Comment:
This should happen at the end, after the docs publish job has run successfully

Reply:
The `publish-reporter-script` adds two comments in the PR. First one comment is related to the start if the job, [link](https://github.com/googleapis/google-cloud-php/pull/6432#issuecomment-1651717491). Second one is related to the completion of the job, [link](https://github.com/googleapis/google-cloud-php/pull/6432#issuecomment-1651726486).